README.md

```diff
@@ -4,7 +4,7 @@ The purpose of this guide is to define standards for all code reviews done by Tr
 
 As a Senior Engineer, code review is one of the most important aspects of what you do and doing it right is instrumental to grooming the right type of engineers and writing effective code in general.
 
-Most of the information provided in this document was adopted from the [Thoughtbot Code Review Guide](https://github.com/thoughtbot/guides/tree/master/code-review).
+Most of the information provided in this document was adopted from the [thoughtbot Code Review Guide](https://github.com/thoughtbot/guides/tree/master/code-review).
```

Comment:
:smile: That is how they spell it, branding I guess. Because I was corrected when I capitalized the "t"